examples/keras_simple.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
Please add this example to the list in [examples/README.md](https://github.com/pfnet/optuna/tree/master/examples#examples-with-ml-libraries).